eclcli/common/utils.py

```diff
@@ -347,6 +347,31 @@ def validate_ipv4(text):
         raise exceptions.CommandError(msg % text)
     return text
 
+def validate_int_range(val, attr_name, min_value=None, max_value=None):
```

Comment:
If you modify this function like this, you can pass it to parser.add_argument as type. You can use other validation functions(validate_ipv4 etc.) as reference.
```suggestion
def validate_int_range(text):
```